giving-example/Views/Home/Result.cshtml

```diff
@@ -0,0 +1,52 @@
+@{
+    ViewData["Title"] = @ViewBag.Status;
+}
+
```

Comment:
In Node I have added a `</br>` here so the page title and the success icon don't overlap